src/tests/suite/daos_test.c

```diff
@@ -241,13 +265,17 @@ main(int argc, char **argv)
 		{"degraded",	no_argument,		NULL,	'd'},
 		{"rebuild",	no_argument,		NULL,	'r'},
 		{"group",	required_argument,	NULL,	'g'},
+		{"csum_type",	required_argument,	NULL,	0x2713},
+		{"csum_cs",	required_argument,	NULL,	0x2714},
+		{"csum_sv",	no_argument,		NULL,	0x2715},
```

Comment:
No, but I didn't want to waste single characters on the checksum options. 0x2713 is the unicode character for a checkmark :-D 
For how infrequent I think it will be used, I thought it okay to require "--csum_type" instead of a single character. I would not recommend trying to use the coded character. To make it easier in the code I could make it a define or constant??